src/Microsoft.AspNetCore.Sockets.Http/HttpConnectionDispatcher.cs

```diff
@@ -365,7 +365,8 @@ private Task ProcessNegotiate(HttpContext context, HttpSocketOptions options, Co
             context.Response.ContentType = "application/json";
 
             // Establish the connection
-            var connection = _manager.CreateConnection();
+            var pipeOptions = new PipeOptions(pauseWriterThreshold: options.PauseWriterThreshold, resumeWriterThreshold: options.ResumeWriterThreshold);
+            var connection = _manager.CreateConnection(pipeOptions);
```

Comment:
What about the other code path where we connect via a websocket directly without negotiating? This is why we need tests 😄